budget/_vars-budget.html

```diff
@@ -3,7 +3,7 @@
 {% set nav_items = [
     (path, 'index', 'Budget and Strategy'),
     (path + 'strategic-plan/', 'strategic-plan', 'Strategic Plan'),
-    (path + 'performance-plan-report/', 'performance-plan-report', 'Performance Plan Report'),
-    (path + 'financial-report/', 'financial-report', 'Financial Report & Update'),
-    (path + 'funding-request/', 'funding-request', 'Funding Request'),
+    (path + 'performance-plan-report/', 'performance-plan-report', 'Performance Plan & Report'),
```

Comment:
Each year is a separate document, but my point is a bit different. Each year there is one document called "Performance Plan and Report". Financial Reports and Updates, however, are separate things: the annual financial report, and the quarterly update.

If we pluralize to "Performance Plans and Reports" we imply that the plan and the report are separate things, even if that's technically correct either way. If we say "Performance Plan and Reports", as if the 's' is tailing the whole phrase, it seems awkward and confusing, and again suggests those are separate things. If we go all singular--and this is just a vagary of the English language--it seems more plausible to read as a single document. It just so happens that we release that document once a year.

I'm prioritizing clarity about what type of document(s) people will find rather than whether there is more than one. I don't like that it's inconsistent, but based on the names, I think it's better.